cogs/economy.py

```diff
@@ -505,11 +505,11 @@ def already_in_list(self, accounts, user):
         line = [reels[0][1], reels[1][1], reels[2][1]]
 
         display_reels = "~~\n~~  " + \
-            reels[0][0] + " " + reels[1][0] + " " + reels[2][0] + "\n"
-        display_reels += ">" + reels[0][1] + " " + \
-            reels[1][1] + " " + reels[2][1] + "\n"
-        display_reels += "  " + reels[0][2] + " " + \
-            reels[1][2] + " " + reels[2][2] + "\n"
+            reels[0][0].format(**slot_emotes) + " " + reels[1][0].format(**slot_emotes) + " " + reels[2][0].format(**slot_emotes) + "\n"
+        display_reels += ">" + reels[0][1].format(**slot_emotes) + " " + \
+            reels[1][1].format(**slot_emotes) + " " + reels[2][1].format(**slot_emotes) + "\n"
+        display_reels += "  " + reels[0][2].format(**slot_emotes) + " " + \
+            reels[1][2].format(**slot_emotes) + " " + reels[2][2].format(**slot_emotes) + "\n"
 
         if line[0] == "{TWO}".format(**slot_emotes) and line[1] == "{TWO}".format(**slot_emotes) and line[2] == "{SIX}".format(**slot_emotes):
```

Comment:
1. you only really need the values for printing
2. you'll probably never win since `line` only has the emote keys, not the values